lib/testing/src/lib/core/pages/login.page.ts

```diff
@@ -163,7 +183,7 @@ export class LoginPage {
 
     goToLoginPage() {
```

Comment:
browser.driver.get is to be used when the web page doesn't have Angular components, but doesn't really make sense after calling browser.waitForAngularEnabled(true)